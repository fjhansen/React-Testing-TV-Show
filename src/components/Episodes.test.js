import axios from 'axios'
import React from 'react'


import * as rtl from "@testing-library/react";

import "@testing-library/jest-dom/extend-expect";
import Episodes from "./Episodes"

test("<Episodes/> Snapshot Full", async () => {
  const wrapper = rtl.render(<Episodes episodes={[]}/>)
  await wrapper.findAllByTestId(/episodes/i)
  

  expect(wrapper.asFragment()).toMatchSnapshot()
  })