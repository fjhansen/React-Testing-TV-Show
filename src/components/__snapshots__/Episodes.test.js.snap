// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Episodes/> Snapshot Full 1`] = `
<DocumentFragment>
  <div
    class="episodes"
    data-testid="episodes"
  />
</DocumentFragment>
`;
